.travis.yml

```diff
@@ -1,5 +1,7 @@
-sudo: false
+sudo: true
 language: ruby
+before_install:
+  - sudo apt-get install -qq augeas-tools augeas-lenses libaugeas-dev
```

Comment:
You can add an apt repo in the same way, but raphink/augeas isn't whitelisted. If they simply check for package names then and you manage to get the repo whitelisted, it could work I think.